sw-emulator/app/src/main.rs

```diff
@@ -269,7 +269,6 @@ fn main() -> io::Result<()> {
 
     let bus_args = CaliptraRootBusArgs {
```

Comment:
Where do you take the argument that ROM is executing in ACTIVE vs PASSIVE mode?

Reply:
I remove it from the bus itself and in hw-model I have a separate way of uploading it. It's also possible to do this in sw-emulator app. I thought it would be appropriate to add the ACTIVE/PASSIVE things in a separate PR in there as it's not strictly related to a DMA driver